opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -42,4 +42,18 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successfull export request.
+  // When the request is only partially successfull (due to e.g. invalid/bad data),
+  // the server MUST populate the details property with a developer-facing error
+  // message along with the number of accepted log records.
+  ExportLogsPartialSuccess details = 1;
```

Comment:
```suggestion
  // The details of a partially successful export request.
  // When the request is only partially successful (due to e.g. invalid/bad data),
  // the server MUST populate the details property with a developer-facing error
  // message along with the number of accepted log records.
  ExportLogsPartialSuccess partial_success = 1;
```

I would prefer to name this field "partial_success" than "details", since the user who sees this data may not read the type name. I want the reader, seeing one of these, to read "I have a partial success", instead of "I have details".

Reply:
Is this field supposed to be unset if the request is fully successful? Or it can be set but accepted_log_records must match the number of records in the request?

Reply:
About the naming: I don't really have a preference, I struggled to find one, so `partial_success` is good for me. I was not happy with `details` as well. 

>Is this field supposed to be unset if the request is fully successful?
Yes, that's what I was going for. If all is well, I would assume it is not useful to dig into any details, since they can be inferred.

Reply:
Changed to `partial_success` (+ grammar) in [4bd818f](https://github.com/open-telemetry/opentelemetry-proto/pull/390/commits/4bd818fb5f743b1c2716a28ada57d242b0193e36)